apps/site-projects/src/components/modules/Resources/index.js

```diff
@@ -1,9 +1,34 @@
 import ProjectResources from "./ProjectResources";
 import { MainResourcesContainer } from "./StyledResources";
 import YourProjects from "./YourProjects";
+import { useUserDataContext } from "../../../context/UserDataContext";
+import { useState } from "react";
 
+function findUserTeams(userId, dataArray) {
+  const userTeams = [];
+
+  for (const data of dataArray) {
+    const team = data.team;
+    const leaders = team.leaders;
+    const members = team.members;
+
+    const leaderMatch = leaders.find(leader => leader.id === userId);
+
+    const memberMatch = members.find(member => member.id === userId);
+
+    if (leaderMatch || memberMatch) {
+      userTeams.push(data);
+    }
+  }
+
+  return userTeams.length > 0 ? userTeams : null;
+}
+
+function Resources({ projects }) {
+  const { userData } = useUserDataContext();
+  const userProjects = findUserTeams(userData.id, projects)

```

Comment:
Hey Helio i read the code really nice work, only here **userProjects** is _null_ and throws an error because it uses @Enjoy2Live 's user context data and for some reason it doesn't return any teams for him, but using @pyxld-kris 's default user data it works and returns the teams array